src/widget/wtimerpopup.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <QLabel>
+#include <QLineEdit>
+#include <QPushButton>
+
+class WCueMenuPopup : public QWidget {
```

Comment:
This was intended to say WTimerPopup here and everywhere else in this file, right?

Reply:
Yes, Thanks for pointing it out.

Reply:
> I am currently trying to put a QHBoxLayout that will house the 3 push buttons.

why not populate a regular QMenu with QActions? much less hazzle, see WCoverArtMenu
also that'll be a no-brainer to integrate with current menu styles.

Reply:
I'll surely have a look thanks a lot! Not very well versed with the QT API